### PR TITLE
Add, remode and edit todo

### PR DESCRIPTION
**Pull request by mntedev**
I completed the following functionalities.2.
 1.hard coded array removed and replaced by Todo class.
2. added modules
3. added save feature
4. added edit  feature
5. added delete feature with sorting
6. todo list saved in local storage
7 screen shoot of the app
image

### CODE REVIEW

File: src/modules/DomUtils.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  element.parentElement.classList.remove('edit');
+};
+const deleteTodo = (element) => {
+  const parent = element.parentElement;
+
+  const index = element.parentElement.id;
+
+  todoList.deleteTodo(parseInt(index, 10));
+  parent.remove();
+};
```
Comment (ShadyShawkat) [inline src/modules/DomUtils.js]: - [ ] Kindly fix the following behavior.

https://user-images.githubusercontent.com/62290545/171694279-cacbb341-80ad-4bcf-95fd-66cca15a1385.mp4

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @worashf, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
- [ ] Please use the following [readme template](https://github.com/microverseinc/readme-template/blob/master/README.md). and please make sure that all of the sections are edited appropriately.
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mntedev** [comment]
I completed the following tasks.
1. corrected functionality
2. corrected sorting after deletion

**Review by OLIPLICHE — APPROVED** [review]
Hi @worashf , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**